queue/src/fiskaltrust.Middleware.Localization.QueueIT/v2/DailyOperations/DailyClosing0x2011.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.ifPOS.v1.it;
+using fiskaltrust.Middleware.Localization.QueueIT.Constants;
+using fiskaltrust.Middleware.Localization.QueueIT.Extensions;
+using fiskaltrust.storage.V0;
+
+namespace fiskaltrust.Middleware.Localization.QueueIT.v2.DailyOperations
+{
+    public class DailyClosing0x2011 : IReceiptTypeProcessor
+    {
+        private readonly IITSSCD _itSSCD;
+        private readonly IJournalITRepository _journalITRepository;
+
+        public ITReceiptCases ReceiptCase => ITReceiptCases.DailyClosing0x2011;
+
+        public DailyClosing0x2011(IITSSCD itSSCD, IJournalITRepository journalITRepository)
+        {
+            _itSSCD = itSSCD;
+            _journalITRepository = journalITRepository;
+        }
+
+        public async Task<(ReceiptResponse receiptResponse, List<ftActionJournal> actionJournals)> ExecuteAsync(ftQueue queue, ftQueueIT queueIt, ReceiptRequest request, ReceiptResponse receiptResponse, ftQueueItem queueItem)
+        {
+            try
```

Comment:
Is there a reason for wrapping the daily operation receipts in a try catch internaly?

I have not yet wrapped my head around he implications in combination with he try catch in the sign processor.
I feel like this is too hard to understand what is being returned if which part of the process fails.

```cs
try
{
    (var response, var actionJournals) = await receiptTypeProcessor.ExecuteAsync(queue, queueIT, request, receiptResponse, queueItem).ConfigureAwait(false);
    if (response.HasFailed())
    {
        return (response, actionJournals);
    }
    return (response, actionJournals);
}
catch (Exception ex)
{
    _logger.LogError(ex, "Failed to process request");
    return await ProcessFailedReceiptRequest(queueIT, queueItem, receiptResponse);
}
```

Reply:
WIP.